etools-content-panel.html

```diff
@@ -89,7 +89,9 @@
         <div class="title">[[title]]</div>
       </paper-toolbar>
       <iron-collapse opened="{{open}}">
-        <slot></slot>
+        <div id="content-wrapper">
+          <content></content>
```

Comment:
According with Polymer 1.7.0 each <content> insertion point must be changed to <slot>.